src/stcal/jump/twopoint_difference.py

```diff
@@ -119,25 +117,41 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
     dnu_flag = dqflags["DO_NOT_USE"]
     jump_flag = dqflags["JUMP_DET"]
 
-    for integ in range(nints):
-
-        log.info(f'Working on integration {integ + 1}:')
-
-        # get data, gdq for this integration
-        dat = dataa[integ]
-        gdq_integ = gdq[integ]
+    # get data, gdq
+    dat = dataa
+    num_flagged_grps = 0
+    # determine the number of groups with all pixels set to DO_NOT_USE
+    for integ in range(dat.shape[0]):
```

Comment:
Shape variables are unpacked above and should be used because of descriptive variable names.